### DISCUSSION TITLE
Rate signifier for double time mod customization on leaderboards

### DISCUSSION THREAD

**Discussion by over-loadcode** [ppy/osu#21250] [Ideas] [answered]
When looking at leaderboards, it seems like there should be a way to tell at a glance what speed increase a double time play was done with. There's a big difference between a top player playing a map at the default value of 1.5x and a player training their speed on the same map at 195bpm instead of 185 (1.05x).

Personally, I think the most "lazer" way to do this would be with a color gradient. The default value of 1.5x could be the default difficulty increase mod color, lower values could lean more towards yellow/orange and higher values could lean more towards red/hot pink. I don't believe that this color spectrum encroaches on any of the already designated colors that heavily.

**Comment by smoogipoo** [comment, marked as answer]
There are already several discussions on this. One that sticks out is https://github.com/ppy/osu/discussions/15262, but also https://github.com/ppy/osu/discussions/14066 and https://github.com/ppy/osu/discussions/19750 are relevant.
